terracumber_config/tf_files/SUSEManager-4.1-NUE.tf

```diff
@@ -173,6 +173,7 @@ module "cucumber_testsuite" {
     }
     build-host = {
       image = "sles15sp2o"
+      name = "min-build"
```

Comment:
build-host, kvm-host, xen-host might be also not necessary...

Reply:
Yes but now I already did the changes 😉